src/entities/returnUser.ts

```diff
@@ -0,0 +1,26 @@
+import "reflect-metadata";
+import { Entity, Column } from "typeorm";
+
+@Entity({ name: "returnUser" })
+export class ReturnUser {
+  @Column()
```

Comment:
tudo bem que o typeorm infere, mas seria mais interessante vc deixar explícito quando nao for varchar